cmd/thanos/compact.go

```diff
@@ -623,15 +643,20 @@ func (cc *compactConfig) registerFlag(cmd extkingpin.FlagClause) {
 		Default("48h").SetValue(&cc.deleteDelay)
 
 	cmd.Flag("compact.enable-vertical-compaction", "Experimental. When set to true, compactor will allow overlaps and perform **irreversible** vertical compaction. See https://thanos.io/tip/components/compact.md/#vertical-compactions to read more."+
```

Comment:
Let's add a space here: 
```suggestion
	cmd.Flag("compact.enable-vertical-compaction", "Experimental. When set to true, compactor will allow overlaps and perform **irreversible** vertical compaction. See https://thanos.io/tip/components/compact.md/#vertical-compactions to read more. "+
```

Reply:
Done in https://github.com/thanos-io/thanos/pull/4239/commits/d0cba248a2914669389d31d06e517327df272b6d